packages/react-badge/src/components/PresenceBadge/usePresenceBadgeStyles.ts

```diff
@@ -0,0 +1,82 @@
+import { ax, makeStyles } from '@fluentui/react-make-styles';
+import { PresenceBadgeState } from './PresenceBadge.types';
+import { useRootStyles as useBadgeStyles } from '../../Badge';
+
+export const usePresenceBadgeRootStyles = makeStyles<PresenceBadgeState>([
+  [
+    null,
+    {
+      padding: 0,
+    },
+  ],
+  [
+    s => s.status === 'busy',
+    theme => ({
+      backgroundColor: theme.global.palette.cranberry.primary,
+      borderColor: theme.global.palette.cranberry.primary,
```

Comment:
These should be using alias colors like `theme.alias.color.cranberry.background3`, not global palette colors.

Reply:
Done